data/distrodefs/fedora/imagetypes.yaml

```diff
@@ -2254,3 +2279,23 @@ image_types:
       - "distro"
       - "customizations.fips"
       - "customizations.locale"
+
+  "pxe-tar":
```

Comment:
Let's call it `pxe-tar-xz` to be consistent with the other image types in this file that are compressed.

I understand this doesn't have a variant so keeping that out of the name is probably fine for now.